packages/github/src/context.ts

```diff
@@ -15,6 +15,8 @@ export class Context {
   workflow: string
   action: string
   actor: string
+  runNumber: string
```

Comment:
Hey @8398a7 , `runNumber` and `runId` are numbers in the [rest api](
https://developer.github.com/v3/actions/workflow-runs/#list-workflow-runs) , we should match that here.

Reply:
Thanks for the review.
I fixed it in 728bcb5.